synth.py

```diff
@@ -29,6 +29,19 @@
   library = gapic.node_library(name, version)
   s.copy(library, excludes=["package.json", "README.md"])
 
+# Expose helper module
+index_file = 'src/index.ts'
```

Comment:
Yes, this one is obviously a ❄️ . We have a plan to think how these exceptions (libraries augmented by extra stuff, like speech, vision, and this one) can be avoided, this is something I will look into in Q1.

Reply:
After chatting with @alexander-fenster and @sofisl , I've reverted the changes to synth.py so that we exclude the handwritten changes from being over written (rather than having custom post-processing steps in synth.py).